assets/queries/cloudFormation/user_missing_group/query.rego

```diff
@@ -27,3 +27,17 @@ CxPolicy[result] {
 		"keyActualValue": "'Resources.Properties.Groups' is empty",
 	}
 }
+
+CxPolicy[result] {
```

Comment:
if `Groups` is defined it can never be `null`. Either an empty or filled list of strings. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-iam-user.html#cfn-iam-user-groups

This policy is not necessary.

Reply:
Understood. Removed that validation.